openjdk-test-common/src/test/resources/workspace/setup-java.bash

```diff
@@ -60,10 +60,11 @@ JAVA_TMP_OPTS=-XX:Temp
 JAVA_HEAP_OPTS=-XX:Heap
 JAVA_GC_OPTS=-XX:GC
 DBG_AGENT=debug
+PROFILER_AGENT=profiler
 JAVA_USER_OPTS=user
 
 source /setup-env.d/30-java-env.bash
-if [ "$(echo $JAVA_OPTS | xargs)" != "-showversion -XX:Temp debug -XX:Heap -XX:GC user" ]; then
+if [ "$(echo $JAVA_OPTS | xargs)" != "-showversion -XX:Temp debug profiler -XX:Heap -XX:GC user" ]; then
```

Comment:
I know this is there for a while now, but why do we need the xargs?

Reply:
I think it was there just to allow easier comparison. Again, this is just used in tests. I would ignore this issue as part of this PR.